d/drupal-unlimited_number/drupal-unlimited_number_ubi_8.5.sh

```diff
@@ -0,0 +1,102 @@
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#   
+# ----------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME=unlimited_number
+CORE_PACKAGE_NAME=drupal
+PACKAGE_URL=https://git.drupalcode.org/project/unlimited_number
+CORE_PACKAGE_URL=https://github.com/drupal/drupal
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-2.0.1}
+
+yum module enable php:7.4 -y
+yum install -y git php php-json php-dom php-mbstring zip unzip gd gd-devel php-gd php-pdo php-mysqlnd
+php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && php composer-setup.php --install-dir=/bin --filename=composer
+
+OS_NAME=`cat /etc/os-release | grep PRETTY_NAME | cut -d '=' -f2 | tr -d '"'`
+#Check if package exists
+if [ -d "$CORE_PACKAGE_NAME" ] ; then
+        rm -rf $CORE_PACKAGE_NAME
+        echo "$CORE_PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+fi
+
+if ! git clone $CORE_PACKAGE_URL $CORE_PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+        echo "$CORE_PACKAGE_URL $CORE_PACKAGE_NAME"
+        echo "$CORE_PACKAGE_NAME  |  $CORE_PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+        exit 1
+fi
+
+cd $CORE_PACKAGE_NAME
+git checkout 8.9.0
+rm -rf composer.lock
+
+composer config allow-plugins true --no-interaction
+if ! composer install --no-interaction; then
+        echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+        exit 1
+fi
+composer require --dev phpunit/phpunit --with-all-dependencies ^7 --no-interaction
+cd modules/
+
+#Check if package exists
+if [ -d "$PACKAGE_NAME" ] ; then
+        rm -rf $PACKAGE_NAME
+        echo "$PACKAGE_NAME  | $PACKAGE_VERSION | $OS_NAME | GitHub | Removed existing package if any"
+fi
+ if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails"
+        exit 1
+fi
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+cd ../../
+cd core/
+
+composer config allow-plugins true --no-interaction
+if ! composer update --no-interaction --ignore-platform-req=ext-gd; then
+    	echo "------------------$PACKAGE_NAME:install_fails---------------------------------------"
+		echo "$PACKAGE_URL $PACKAGE_NAME"
+        echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Install_Fails"
+    	exit 1
+else 
```

Comment:
The else block is not required here as it will block the execution of Test step below. If you still want to keep the else block, do not write 'exit 0' at line no 87, so that it can execute line no 90.